docs/ota.md

```diff
@@ -7,27 +7,873 @@ The messages are encoded as json payloads, and you can validate any payload by u
 Ruters repositories.  
 
 MQTT is used as the transport mechanism for all OTA messages. MQTT is a simple pub-sub based messaging protocol, that works on top of TCP/IP as well as on websockets. 
+Ruter has agreed with the operators to use only JSON in MQTT messages and not support Protobuf. The contents of the messages will therefore be defined by JSON schemas and these will be made available to the operators.
```

Comment:
Copied directly over, but somewhat redundant considering the paragraph above. Should we remove this/some of it?